components/ide/code/leeway.Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN sudo apt-get update \
     && sudo apt-get clean -y \
     && rm -rf /var/lib/apt/lists/*
 
-ENV GP_CODE_COMMIT ae8dc44299153ca820ac9e13cc9def5bb6baafd1
+ENV GP_CODE_COMMIT 334e0610dd1bb73487812b417b113d6a2b96e1b1
```

Comment:
**praise**: Maybe this is by design for the Code editor but I love how clicking the Gitpod link on the status bar opens up all the available Gitpod commands. 🎉 

<img width="591" alt="Screenshot 2021-01-28 at 9 14 06 PM" src="https://user-images.githubusercontent.com/120486/106187049-c6f34400-61ad-11eb-8037-6be5037787b7.png">